### ISSUE TITLE
Question regarding the benchmark and TILs

### ISSUE THREAD

**Issue by Reasat** [CancerDataScience/HistomicsML-TA#72]
Hi 
I have a few questions regarding the benchmark in the [HIstomicsML2 paper](https://arxiv.org/ftp/arxiv/papers/2001/2001.11547.pdf).

`For the BRCA dataset, a series of 10 classifiers was generated as a comparison to evaluate the benefit of active learning training in HistomicsML2.0. Each training slide in the BRCA dataset has an associated ground truth annotation, and so we formed 10 training sets by sampling superpixels from these images and using the ground-truth annotations for labeling.`

The referred dataset has only [ROI annotations](https://github.com/CancerDataScience/CrowdsourcingDataset-Amgadetal2019/blob/master/meta/roiBounds.csv) and not the whole slide annotations. So, is it safe to assume that when the training set  was made for the benchmark classifiers, samples from only the ROI were taken (and not from the whole slide?)

`For the purpose of this work, we considered all small mononuclear cells with lymphocyte-like morphology to be TILs. Dense purely plasma cell infiltrates were not included in TIL regions, but admixtures of lymphocytes and plasma cells were counted as lymphocytic infiltrates. Artifacts, normal epidermal or dermal structures, and nearby lymphoid aggregates were excluded from the regions of interest.`

Are the classes derived from this [labeling scheme](https://github.com/CancerDataScience/CrowdsourcingDataset-Amgadetal2019/blob/master/meta/gtruth_codes.tsv)? Could you tell me which labels correspond to TILs? Is it only class 3 (lymphocytic_infiltrate)? Are other classes also included in the TILs?

Thanks in advance!

**Comment by cooperlab** [comment]
To answer your first question, all ground truth used in testing or validation and all sampled labels used in training originate from the ROI annotation dataset published in Bioinformatics.

To answer your second question, the paragraph you cite refers to the cutaneous melanoma dataset. This is separate from the breast cancer dataset that you are referring to in your question. For the breast cancer TIL experiments we used codes 3 and 10 from this table to represent the positive class.

**Comment by Reasat** [comment]
Yes, you are right, I cited the SKCM paragraph (should have cited the BRCA part). Thanks for answering the questions!
